gpflow/training/monitor/__init__.py

```diff
@@ -1,4 +1 @@
-from .actions import *
-
-print("Warning: `monitor` is still in beta. It is slightly buggy, will change, "
-      "and is not tested properly in the unittests.")
+from .monitor import *
```

Comment:
Super bad! :) https://stackoverflow.com/questions/2386714/why-is-import-bad

Reply:
I deleted the __init__.py altogether.